README.md

```diff
@@ -53,13 +53,33 @@ persist('some', someStore, {
   - **key** *string* The key of your storage engine that you want to persist to.
   - **store** *MST store* The store to be persisted.
   - **options** *object* Additional configuration options.
-    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage* [localForage](https://github.com/localForage/localForage)-style storage API. localStorage for Web (default), AsyncStorage for React Native.
+    - **storage** *[localForage](https://github.com/localForage/localForage) / AsyncStorage / localStorage*
+      Any Storage Engine that has a Promise-style API similar to [`localForage`](https://github.com/localForage/localForage).
+      The default is `localStorage`, which has a built-in adapator to make it support Promises.
```

Comment:
ahhk, there's a typo here, should be "adaptor".

need some spell-check in my editor to catch these things 😅